backend/eth/client.go

```diff
@@ -111,14 +111,16 @@ func (c *Client) GetCode(
 	ctx context.Context,
 	req backend.GetCodeRequest,
 ) (backend.GetCodeResponse, errors.Err) {
-	v, err := c.tracker.Instrument(getCode, func() (interface{}, error) {
-		return c.getCode(ctx, req)
-	})
+	timer := c.metrics.RequestTimer(getCode)
```

Comment:
I believe the convention is to snake_case Prometheus labels, if you can convert. [strcase](https://github.com/iancoleman/strcase) is a nice package for it.